conf.c

```diff
@@ -36,6 +36,8 @@ struct fuse_opt kibosh_command_line_options[] = {
      KIBOSH_CONF_OPT("--target %s", target_path, 0),
      KIBOSH_CONF_OPT("--control-mode=%o", control_mode, 0600),
      KIBOSH_CONF_OPT("--control-mode %o", control_mode, 0600),
+     KIBOSH_CONF_OPT("--random-seed=%d", random_seed, 0),
```

Comment:
should it be initialized to `time()` so that it's unpredictable by default?

Reply:
Originally the default seed was set to time(0) in main.c. I think it makes more sense to set it here. Edited as suggested.